src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3160,11 +3160,18 @@ private void AfterLeftChildHasBeenVisited(BoundExpression leftOperand, Conversio
                     ParameterSymbol parameter,
                     TypeWithState operandType)
                 {
+                    TypeWithAnnotations targetTypeWithNullability = parameter.TypeWithAnnotations;
```

Comment:
>targetTypeWithNullability [](start = 40, length = 25)

Nit: Not a fan of the naming. `Nullability` in this file generally means either annotation state or flow state. This is really `possiblyLiftedTargetType`.

Reply:
The name matches parameter name of the VisitConversion method that is called below.

---
In reply to: [405826407](https://github.com/dotnet/roslyn/pull/43165#discussion_r405826407) [](ancestors = 405826407)